src/TypeSpec.Extension/Emitter.Csharp/src/lib/converter.ts

```diff
@@ -96,6 +96,31 @@ function fromSdkType(
     return {} as InputType;
 }
 
+// this function is only for the case when we get a type from a parameter, because in typespec, the parameters are
```

Comment:
```suggestion
// this function is only for the case when we get a type from a property, because in typespec, the properties are
```

Reply:
because this is handling an operation parameter, we have to add this comment to specify, this is not a typo.

Reply:
The function called `fromSdkModelPropertyType` and it handle `SdKModelPropertyType`, so it is confused that it handle parameters in comments.

Reply:
it is confused because TCGC named this type in this way. We cannot control it and this is the purpose we add this comment to emphasize that despite the name says "property", but we are handling parameters.
Also this should go away when we have getAllOperations